host/xtest/regression_4000.c

```diff
@@ -4517,11 +4516,9 @@ static void xtest_test_keygen_dh(ADBG_Case_t *c, TEEC_Session *session)
 				       key_types[n].subprime_len);
 		}
 
-		if (!ADBG_EXPECT_TRUE(c,
-			generate_and_test_key(c, session, TEE_TYPE_DH_KEYPAIR,
-				*key_types[n].private_bits,
-				key_types[n]. key_size, params, param_count)))
-			break;
+		generate_and_test_key(c, session, TEE_TYPE_DH_KEYPAIR,
+			*key_types[n].private_bits,
+			key_types[n]. key_size, params, param_count);
```

Comment:
Here and below the result of `generate_and_test_key()` is not tested anymore. Why?